packages/portal/spec-renderer/src/components/SpecOperationsList.vue

```diff
@@ -32,7 +32,7 @@
           v-for="section in sectionHeadings"
           :key="section"
           class="section-wrapper"
-          :data-testid="`spec-operations-list-section-${section.toLowerCase().replaceAll(' ', '-')}`"
+          :data-testid="`spec-operations-list-section-${String(section).toLowerCase().replaceAll(' ', '-')}`"
```

Comment:
I don't understand what this fixes? If `section` is undefined, you're going to be trimming a string of `'undefined'`

Reply:
It shouldn't be undefined as in `sectionHeadings` computed checks whether it's not falsy here: https://github.com/Kong/public-ui-components/blob/5c52d1fa5ac3ad6708911374d2794f35000e06f3/packages/portal/spec-renderer/src/components/SpecOperationsList.vue#L229

Reply:
I'm not sure how this error happened but it happened. I reworked my fix a bit